### PR TITLE
Reduced window minWidth and minHeight

### PR DESCRIPTION
**Pull request by anthonyattard**
Reduced window minWidth and minHeight so that it can accomodate for split screen.

### CODE REVIEW

**Comment by hackjutsu** [comment]
Would you mind posting a screenshot to illustrate the difference?

**Comment by anthonyattard** [comment]
Here are some screenshots. As you can see in before.png the window min size is too large, causing window overlap.
before
before-min-size
after
after-min-size

File: main.js [hunk 1]
```diff
@@ -68,8 +68,8 @@ function createWindow (autoLogin) {
     height: mainWindowState.height,
     x: mainWindowState.x,
     y: mainWindowState.y,
-    minWidth: 1000,
-    minHeight: 700,
+    minWidth: 636,
+    minHeight: 609,
```
Comment (hackjutsu) [inline main.js]: Interesting, but these look like magic numbers for me. It would be great to set them based on the research of popular screen sizes.

Comment (makecloud) [reply main.js]: good job

Comment (anthonyattard) [reply main.js]: I chose those numbers based on the minimum size that looks good for the way lepton is layed out (to fit languages, pinned, tags, etc).

Comment (hackjutsu) [reply main.js]: Interesting, let's merge and iterate on these numbers. Thanks :)

Comment (anthonyattard) [reply main.js]: :)

**Pull request merged**